tests/Export/Adapters/KeywordsAdapterTest.php

```diff
@@ -17,26 +18,33 @@ class KeywordsAdapterTest extends TestCase
     use AdapterHelper;
     use ProductHelper;
 
-    public function testKeywordsContainsTheKeywordsOfTheProduct(): void
+    public function testKeywordsContainsTheKeywordsOfTheProductForSelectedLanguage(): void
     {
         $expectedKeywords = [new Keyword('keyword1'), new Keyword('keyword2')];
-        $keywordsEntities = [$this->getKeywordEntity('keyword1'), $this->getKeywordEntity('keyword2')];
+        $keywordsEntities = [
+            $this->getKeywordEntity('keyword1', '2fbb5fe2e29a4d70aa5854ce7ce3e20b'),
+            $this->getKeywordEntity('keyword2', '2fbb5fe2e29a4d70aa5854ce7ce3e20b'),
+        ];
```

Comment:
You are not testing, if a keyword with a different languageId is ignored. Please add a third keyword with a random UUID.